Samples/WinMLSamplesGallery/WinMLSamplesGallery (Package)/Package.appxmanifest

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="42365107-cbb5-412e-80ac-554e5225ab17"
+    Publisher="CN=sheil"
+    Version="1.0.0.0" />
+
+  <Properties>
+    <DisplayName>WinMLSamplesGallery (Package)</DisplayName>
+    <PublisherDisplayName>sheil</PublisherDisplayName>

```

Comment:
let's use a different publisher name

Reply:
Should the publisher name be Windows ML?